pom.xml

```diff
@@ -1,169 +1,225 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-  <groupId>de.roskenet</groupId>
-  <artifactId>springboot-javafx-support</artifactId>
-  <version>1.3.21-SNAPSHOT</version>
-  <packaging>jar</packaging>
+	<groupId>net.springjfx</groupId>
```

Comment:
I like the name springjfx but I will add it as Zalando OSS project in the near future that willl have different group and artifact names anyways.
See https://zalando.github.io/
So I won't apply those changes now.